dbms/src/Flash/Mpp/ExchangeReceiver.h

```diff
@@ -184,6 +190,7 @@ class ExchangeReceiverBase
     std::condition_variable cv;
     /// should lock `mu` when visit these members
     Int32 live_connections;
+    Int32 live_local_connections;
```

Comment:
Because there will only be one local tunnel, it can be replaced with `bool is_local_connectoin_alive = false`.

Reply:
> 

I think that counting number of local tunnels could handle more situation because maybe more local tunnels will be created in the furture.

Reply:
It is only a bug if there is more than one local tunnel.
Because these source tasks that executed on the same machine can be combine into one.

Reply:
> It is only a bug if there is more than one local tunnel. Because these source tasks that executed on the same machine can be combine into one.

okk, I will fix it